webpack/set-webpack-public-path-plugin/README.md

```diff
@@ -126,6 +126,10 @@ Note that the existing value of the variable already ends in a slash (`/`).
 If true, find the last script matching the regexVariable (if it is set). If false, find the first matching script.
 This can be useful if there are multiple scripts loaded in the DOM that match the regexVariable.
 
+#### `skipDetection = false`
+
+If true, always include the code snippet to detect the public path regardless of whether chunks or assets are present.
```

Comment:
>are [](start = 106, length = 3)

Why would someone want to do this?

Reply:
Addressed in the other comment.

---
In reply to: [194925316](https://github.com/Microsoft/web-build-tools/pull/685#discussion_r194925316) [](ancestors = 194925316)